src/main/java/org/jitsi/jicofo/jigasi/TranscriberManager.java

```diff
@@ -201,14 +201,26 @@ private void startTranscribing()
             return;
         }
 
+        selectTranscriber(2, null);
+    }
+
+    /**
+     * Sends the message to the selected jigasi (from brewery muc).
```

Comment:
What message does this send? Does it also do the selection logic like the name suggests?